package.json

```diff
@@ -52,11 +53,12 @@
     "body-parser": "^1.18.2",
     "cassandra-driver": "^3.5.0",
     "detect-port": "^1.2.2",
+    "drill-client": "^0.1.2",
     "errorhandler": "^1.4.0",
     "express": "^4.16.3",
     "express-session": "^1.15.6",
     "hdb": "^0.15.4",
-    "helmet": "^3.12.0",
+    "helmet": "^3.15.0",
```

Comment:
Is it possible to revert the dependency updates in package.json (and package-lock.json)?

I've split the front-end out of the the primary package.json and would like to avoid merge conflicts if possible